packages/grid/_modules_/grid/hooks/utils/useGridApiRef.ts

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { GridApiRef } from '../../models/api/gridApiRef';
 import { GridApi } from '../../models/api/gridApi';
+import { GridState } from '../../models/gridState';
 import { EventManager } from '../../utils/EventManager';
 
 // Public developers facing overload
```

Comment:
Maybe we should think of a better name of the file that exports public API 🤔

Reply:
The `utils` folder contains stuff that can be used on several places (by opposition with the `core` or `feature` folder where you only instantiate each hook once).

But I agree that putting 100%-internal-stuff in it next to exported utility hooks is not ideal.
We could split the folder in two.